master/static/srv/get_checkpoint.sql

```diff
@@ -0,0 +1,17 @@
+SELECT
```

Comment:
thought: These sql queries doing similar things is a little unfortunate -- but I imagine the new API project can handle cleaning this up

Reply:
That is correct, there is no clear way of doing this currently.